laser_encoders/models.py

```diff
@@ -167,12 +167,17 @@ def batch(tokens, lengths, indices):
         if nsentences > 0:
             yield batch(batch_tokens, batch_lengths, batch_indices)
 
-    def encode_sentences(self, sentences):
```

Comment:
Could you please write a testcase for this?

Reply:
Ok, Will do that.

Reply:
> Could you please write a testcase for this?

Could you clarify the scope of testing desired for the new normalize_embeddings feature? 
I intend to cover both scenarios where `normalize_embeddings` is set to `True` and `False`. Specifically, I'll verify that the embeddings are correctly L2-normalized when the flag is True and remain unchanged when it is False. Are there any additional cases or behaviors we should test for to ensure the feature's correctness?"

Reply:
No, I think this is enough.